sunnypilot/selfdrive/controls/lib/auto_lane_change.py

```diff
@@ -104,6 +104,12 @@ def update_lane_change(self, blindspot_detected: bool, brake_pressed: bool) -> N
     if brake_pressed and not self.prev_brake_pressed:
       self.prev_brake_pressed = brake_pressed
 
+    # If AutoLaneChangeMode is OFF, skip all lane change logic
+    if self.lane_change_set_timer == AutoLaneChangeMode.OFF:
+      self.auto_lane_change_allowed = False
+      self.lane_change_wait_timer = 0.0  # Reset wait timer
+      return
+
```

Comment:
With this addressed: https://github.com/sunnypilot/sunnypilot/pull/826/files#r2045281372

This condition now becomes unreachable, we can remove.